examples/upload-files/README.MD

```diff
@@ -0,0 +1,25 @@
+# Select Files To Upload
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+TestCafe cannot test native browser dialogs, including those during file selection. For this reason, TestCafe utilizes another approach to file selection. This example shows how to select files to populate a `type="file"` input.
+
+The sample page includes a `<form>` that contains:
+
+* `<input>` element that holds the selected files
+* the `submit` button that sends the files to the server.
+
+During the test the `t.setFilesToUpload` method selects the files to send through the `<input>`. A `submit` button click sends the files. The server responds with a list of uploaded files, which is evaluated with the `t.expect.eql` Method.
```

Comment:
During the test the `t.setFilesToUpload` method set the files for the `<input type=file`. A `input type=submit` initiates the sending files to the server. The server responds with a list of uploaded files, which is checked with the `t.expect.eql` method.

Reply:
> During the test the `t.setFilesToUpload` method sets the files for the `<input type="file">`. A click on the `<input type="submit">` initiates the file upload to the server. The server responds with a list of uploaded files, which is checked with the `t.expect.eql` Method.

Reply:
Ok